.gitlab/pipeline/test.yml

```diff
@@ -17,6 +17,7 @@ test-linux-stable:
     # Enable debug assertions since we are running optimized builds for testing
     # but still want to have debug assertions.
     RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
+    RUST_LOG: "polkavm=trace"
```

Comment:
We will remove later. We just need to debug why the test is failing on CI but not locally.

Reply:
You might want to test locally using the docker image, should be faster to debug?

Reply:
Last time I checked our docker images didn't support Apple Silicon. Did that change?

Reply:
ah right that could be I usually test that from my linux box

Reply:
Yeah I have a linux box I can test that on, too. Will do that. Takes too long on the CI.